lib/ios-sim.ts

```diff
@@ -15,6 +15,15 @@ var fiber = Fiber(() => {
 
 fiber.run();
 
+function getSimulator(): IFuture<ISimulator> {
+	return (() => {
+		let libraryPath = require("./iphone-simulator");
+		let obj = new libraryPath.iPhoneSimulator();
+		let simulator = obj.createSimulator().wait();
```

Comment:
Optionally, you can just `return obj.createSimulator();` and remove the return xxx.future()().